### PR TITLE
Updated broken links in ./README.md

### PR DESCRIPTION
**Pull request by gfoidl**
Please check if they point to the correct destination now. I used the ones that seemed most plausible to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ We welcome contributions! Many people all over the world have helped make this p
...
 
 * If you want to contribute a fix or improvement, you'll want to check out these links
 
-    * Requirements to build this repo ([Linux](docs/building/linux-requirements.md), [macOS](docs/building/macos-requirements.md), [Windows](docs/building/windows-requirements.md))
-    * What kind of contributions you might want to make [in the runtime](docs/coreclr/project-docs/contributing.md) and [in the libraries](docs/libraries/project-docs/contributing.md)
-	* [How you can make a change happen!](docs/coreclr/project-docs/contributing-workflow.md)
-	* How to build and test your change [in the runtime](docs/coreclr/project-docs/developer-guide.md) and [in the libraries](docs/libraries/project-docs/developer-guide.md)
+    * Requirements to build this repo ([Linux](docs/workflow/linux-requirements.md), [macOS](docs/workflow/macos-requirements.md), [Windows](docs/workflow/windows-requirements.md))
+    * What kind of contributions you might want to make [in the runtime](docs/coreclr/project-docs/contributing.md) and [in the libraries](docs/project/changing-corelib.md)
```
Comment (gfoidl) [inline README.md]: The link for "runtime" still points to nothing -- couldn't find anything that matches.

Comment (jkotas) [inline README.md]: `docs/project/changing-corelib.md` document should be deleted. The workflow that it describes is not relevant in the consolidated repo.

Comment (jkotas) [reply README.md]: It maybe best to delete the line.

File: README.md [hunk 2]
```diff
@@ -24,14 +24,14 @@ We welcome contributions! Many people all over the world have helped make this p
 
 * If you have a question or idea, [file a new issue](https://github.com/dotnet/runtime/issues/new).
 
-	* Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
+    * Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
 
 * If you want to contribute a fix or improvement, you'll want to check out these links
```
Comment (ViktorHofer) [inline README.md]: I think we should delete this whole section. We have a CONTRIBUTING.MD file in the repo root for that (which I admit needs to cleaned up).

Comment (gfoidl) [reply README.md]: Or just point to the contributing.md, like
```
## How can I contribute?

We welcome contributions! Many people all over the world have helped make this project better.

For further information see [Contributing](CONTRIBUTING.md).
```
?

Comment (ViktorHofer) [reply README.md]: That sounds good

**Review by jaredpar — APPROVED** [review]

**Comment by danmosemsft** [comment]
I think this needs retaining, as it doesn't seem to exist elsewhere

> 	* Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.

In general I think we need someone to make an all-up plan for our markdown files, as mentioned elsewhere, to make them consistent, remove dead and duplicate stuff etc.

**Comment by gfoidl** [comment]
Agreed.

My concern / motivation for this PR is the broken links in 
> * Requirements to build this repo (Linux, macOS, Windows)

which were the first things I read on this repo. I assume most contributors will check this also. It's not hard to find the right information when quite familiar with the old repos, but especially for new visitors it may look strange.

As a quick fix we should bring the "landing page" in a state that "works". Later on all the doc-files can be updated to reflect the new repo-structure / links.

---

@danmosemsft this would fit to https://github.com/dotnet/runtime#filing-issues?

File: README.md [hunk 3]
```diff
@@ -22,16 +22,7 @@ Official Starting Page: https://dotnet.microsoft.com/
 
 We welcome contributions! Many people all over the world have helped make this project better.
 
-* If you have a question or idea, [file a new issue](https://github.com/dotnet/runtime/issues/new).
-
-	* Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
-
-* If you want to contribute a fix or improvement, you'll want to check out these links
-
-    * Requirements to build this repo ([Linux](docs/building/linux-requirements.md), [macOS](docs/building/macos-requirements.md), [Windows](docs/building/windows-requirements.md))
```
Comment (pgovind) [inline README.md]: Are we sure we want to delete this entire section? The `Contributing.md` doc doesn't have instructions to build the repo. Maybe delete the `If you have a question or idea...` section but keep the `If you want to contribute a fix...` section?

Comment (ViktorHofer) [reply README.md]: yes we should delete this and in CONTRIBUTING.md point to the build instructions: https://github.com/dotnet/runtime/blob/master/docs/workflow/README.md

Comment (pgovind) [reply README.md]: @gfoidl : Do you mind if I add a commit to this patch updating some more outdated links?

**Comment by gfoidl** [comment]
> Do you mind if I add a commit to this patch updating some more outdated links?

@pgovind you can update this PR as you want. (or close it and make a new one with more links fixed).

The landing page (i.e. ./Readme.md) should be in good state asap, otherwise it looks somewhat odd for such a great repository :wink:

**Comment by ViktorHofer** [comment]
I'm merging this now as is. Please submit a new PR for what we discussed here https://github.com/dotnet/runtime/pull/277#discussion_r350980809.

**Comment by ViktorHofer** [comment]
Thanks a lot @gfoidl

**Pull request merged**